deepspeed/comm/comm.py

```diff
@@ -264,18 +263,18 @@ def reduce_scatter_fn(output_tensor,
     global cdb
     global has_warned_reduce_scatter
     assert cdb is not None and cdb.is_initialized(), 'DeepSpeed backend not set, please initialize it using init_process_group()'
-    if cdb.has_reduce_scatter_base:
-        return reduce_scatter_base(output_tensor,
-                                   tensor,
-                                   op=op,
-                                   group=group,
-                                   async_op=async_op,
-                                   prof=prof,
-                                   debug=debug)
+    if cdb.has_reduce_scatter_tensor():
```

Comment:
Same comment on backwards compatibility as for all_gather

Reply:
same comment as above